src/components/desktop/Header.js

```diff
@@ -48,28 +76,89 @@ const StyledHeader = styled.header`
           font-family: inherit;
           font-weight: normal;
           cursor: pointer;
+          background-color: ${({ theme }) => theme.background};
         }
 
         button,
         &.active {
-          background-color: ${({ theme }) => theme.background};
           color: ${({ theme }) => theme.orange};
         }
       }
     }
   }
 `
+const Dropdown = ({ className, children, text }) => (
+  <div className={className}>
+    <div className="dropbtn">
+      {text} <CaretDown height="8px"> </CaretDown>
+    </div>
+    <div className="dropdown-content">{children}</div>
+  </div>
+)
+
+const StyledDropdown = styled(Dropdown)`
+  display: inline-block;
+
+  .dropbtn {
+    background-color: ${({ theme }) => theme.background};
+    color: ${({ theme }) => theme.text};
+    padding: 16px;
+    display: inline;
+    border: none;
+    height: 100%;
+    width: 100%;
+  }
+
+  &:hover .dropdown-content {
+    display: block;
+  }
+
+  .dropdown-content {
+    display: none;
+    position: absolute;
+    margin-top: 15px;
+    background-color: ${({ theme }) => theme.background};
+    border-radius: 0px 0px 6px 6px;
+    box-shadow: rgba(0, 0, 0, 0.05) 0 15px 15px;
+    text-align: center;
+    min-width: 100%;
+    left: 0;
+
+    a {
+      color: black;
+      padding: 12px 16px;
+      text-decoration: none;
+      display: block;
+    }
+  }
+`
 
 // TODO: Clean up file structure (i.e. logo_white.svg) from ./public
 const Header = () => {
   const { t } = useTranslation()
   return (
     <StyledHeader>
-      <img src="/logo_white.svg" alt="Falling Fruit Logo" />
+      <img src="/logo_orange.svg" alt="Falling Fruit Logo" />
       <nav>
         <ul>
-          <li className="active">{t('Map')}</li>
-          <li>{t('About')}</li>
+          <li>
+            <NavLink to="/map" activeClassName="active">
+              {t('Map')}
+            </NavLink>
+          </li>
+          <li>
+            <NavLink to="/page2" activeClassName="active">
+              {t('Page 2')}
+            </NavLink>
+          </li>
+          <li>
+            <StyledDropdown text="About">
```

Comment:
Translations should be available for all these links in the dropdown and also on the `text` prop we're passing into our Dropdown component

e.g. it would be odd if my whole app was in French, except for the word 'About' and everything under this dropdown, right? 😅